### PR TITLE
Improvement: Make submodules selectable in document settings and static routes

### PR DESCRIPTION
**Pull request by ThomasKeil**
Submodules, if present, can now be selected in document settings and static routes with a selectbox instead of entering the name by hand, additionally the controller select boxes are updated accordingly.

### CODE REVIEW

File: pimcore/static/js/pimcore/settings/staticroutes.js [hunk 1]
```diff
@@ -181,7 +191,17 @@ pimcore.settings.staticroutes = Class.create({
                     }),
                     triggerAction:"all",
                     displayField:'name',
-                    valueField:'name'
+                    valueField:'name',
+                    listeners:{
+                        "focus":function (el) {
+                            console.log();
```
Comment (brusch) [inline pimcore/static/js/pimcore/settings/staticroutes.js]: please remove this

Comment (brusch) [inline pimcore/static/js/pimcore/settings/staticroutes.js]: https://github.com/pimcore/pimcore/commit/293b95a2c608f7db2299367d0b17a1979453b2f3 ;-) 
console.log()

Comment (ThomasKeil) [inline pimcore/static/js/pimcore/settings/staticroutes.js]: *facepalm* I missed that one

Comment (brusch) [inline pimcore/static/js/pimcore/settings/staticroutes.js]: no problem, thanks for the nice PR!

**Pull request merged**